PREREQUISITE.md

```diff
@@ -0,0 +1,248 @@
+# Prerequisite
+
+You need to do two actions.
+
+1. Add euphony to your project
+2. Edit your AndroidManifest.xml to permit euphony to record audio.
```

Comment:
Let's put a link in the main table of contents!

```
1. [Add euphony to your project](#import)
2. [Edit your AndroidManifest.xml to permit euphony to record audio.](#permission)
```

Reply:
Nice work! Thanks for your efforts